op-guide/build.md

```diff
@@ -0,0 +1,145 @@
+# Build
+
+## Supported platforms
+
+The following table lists TiDB support for common architectures and operating systems. 
+
+|Architecture|Operating System|Status|
+|------------|----------------|------|
+|amd64|Linux Ubuntu (14.04+)|Stable|
+|amd64|Linux CentOS (7+)|Stable|
+|amd64|Mac OSX|Experimental|
+
+## Prerequisites
+
++ Go [1.5+](https://golang.org/doc/install)
++ Rust [nightly version](https://www.rust-lang.org/downloads.html)
++ GCC 4.8+ with static library
+
+### Install GO
+
+#### Linux
```

Comment:
I will add a check requirement script instead. 
It will check whether user has correct building environment and will install missing ones.